pyproject.toml

```diff
@@ -56,6 +56,7 @@ onnxruntime = [
   { version = ">=1.7.0,<1.20.0", optional = true, markers = "python_version < '3.10'" },
   { version = "^1.7.0", optional = true, markers = "python_version >= '3.10'" }
 ]
+pubmed-parser = "^0.5.1"
```

Comment:
This package has a questionable license, because it depends on `unidecode` which is licensed as GPL.